{
   "Title": "Denial of Service attack",
   "Timestamp": 1486290968,
   "Risk": 1,
   "CVSS3": {
     "score": 5.0,
     "vector": "AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:N/A:L"
   },
   "CWE": {
     "id": 674,
     "name": "Uncontrolled Recursion"
   },
   "HackerOne": 174524,
   "Affected":[
      {
         "Version":"10.0.2",
         "CVE":"CVE-2017-0886",
         "Operator":"<"
      },
     {
       "Version":"9.0.55",
       "CVE":"CVE-2017-0886",
       "Operator":"<"
     }
   ],
   "Description":"Due to an error in the application logic an authenticated adversary may trigger an endless recursion in the application leading to a potential Denial of Service.",
   "ActionTaken": "The code path leading to the endless recursion is now properly handled.",
   "Acknowledgment":[
      {
         "Name": "secator",
         "Email": "info@secator.com",
         "Website": "https://secator.com/",
         "Reason": "Vulnerability discovery and disclosure."
      }
   ],
   "Resolution": "It is recommended that all instances are upgraded to Nextcloud 9.0.55 or 10.0.2."
}
